buildspec.yml

```diff
@@ -0,0 +1,69 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - apk update
+      - apk add python3
+      - apk add python3-dev
+      - apk add py3-setuptools
+      - pip3 install awscli
+      - python3 --version
+      - terraform --version
+      - START_DIR=$(pwd)
+      - cd $START_DIR/modules/rdcb
```

Comment:
might be easier to define each module name in an array and loop over the contents. not saying its necessary but it would help make things more DRY

Reply:
Sounds good.  I'll add a loop